distribution/src/main/assembly/windows-bin.xml

```diff
@@ -30,6 +30,12 @@
   <includeBaseDirectory>true</includeBaseDirectory>
 
   <fileSets>
+
+    <fileSet>
+      <outputDirectory>lib/metrics.statsd</outputDirectory>
```

Comment:
This results in a subdirectory metrics.statd being created inside {gateway_home}/lib with further subdirectories bin, conf and lib. I do not think this is the right place for this since the gateway/lib directory is supposed to contain just jar files. I think a better location would be just put it directly under the Gateway home, which I think could be done by changing to
```
<outputDirectory>metrics.statsd</outputDirectory>
```
We'll need to see with PM (Robin) ultimately on this, but for now I think this is a reasonable proposal to go with.